JSDemos/Demos/TreeList/UsingFilterRow/description.md

```diff
@@ -1 +1,5 @@
-This demo shows how you can filter data using the filter row. Its cells allow filtering by&nbsp;individual columns. Select or&nbsp;type a&nbsp;value into a&nbsp;cell of&nbsp;the filter row. If&nbsp;this value is&nbsp;found in&nbsp;any of&nbsp;the rows, the TreeList displays the full hierarchy down to&nbsp;them, but not further. In&nbsp;addition, you can choose the current filter operation from the drop-down menu that opens when you hover over the magnifying glass icon.
\ No newline at end of file
+The filter row of a certain column allows users to enter a query and filter rows by that value. To enable this behavior, set **filterRow**.[visible](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/filterRow/#visible) to **true**. This feature can be combined with the [header filter](https://js.devexpress.com/Demos/WidgetsGallery/Demo/TreeList/UsingHeaderFilter).
+
+Each cell in the filter row contains a magnifying glass icon that opens a drop-down list with available filters. These filters and their initial setting are defined by the column's [data type](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#dataType). You can use the [selectedFilterOperation](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#selectedFilterOperation) property to specify a custom initial setting.
```

Comment:
1. Maybe we can use imperative mood?

1. >a custom initial setting

Why do you use the singular form here?

```suggestion
You can click an icon in a filter row cell to open a drop-down list with available filters. These filters and their initial setting are defined by the column's [data type](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#dataType). You can use the [selectedFilterOperation](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#selectedFilterOperation) property to specify custom initial settings.
```